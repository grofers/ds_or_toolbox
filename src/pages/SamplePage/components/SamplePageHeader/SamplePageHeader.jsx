import React from "react";

const SamplePageHeader = () => <section>Sample Header</section>;

export default SamplePageHeader;
