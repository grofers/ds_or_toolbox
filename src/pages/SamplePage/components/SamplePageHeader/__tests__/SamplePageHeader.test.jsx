import React from "react";
import { render } from "@testing-library/react";
import SamplePageHeader from "../SamplePageHeader";

describe("SamplePageHeader", () => {
  it("should match the snapshot on page load", () => {
    const { asFragment } = render(<SamplePageHeader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
