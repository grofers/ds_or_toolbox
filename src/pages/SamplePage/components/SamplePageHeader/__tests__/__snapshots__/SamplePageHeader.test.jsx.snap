// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SamplePageHeader should match the snapshot on page load 1`] = `
<DocumentFragment>
  <section>
    Sample Header
  </section>
</DocumentFragment>
`;
